core/vm/evm.go

```diff
@@ -42,7 +42,7 @@ func (evm *EVM) precompile(addr common.Address) (PrecompiledContract, bool) {
 	var precompiles map[common.Address]PrecompiledContract
 	switch {
 	case evm.chainRules.IsPrague:
-		precompiles = PrecompiledContractsBerlin
+		precompiles = PrecompiledContractsByzantium
```

Comment:
This is the magical line change that fixes the problem. The above file is another place where I saw a similar type of configuration but is only used in tests. Changed it there too to be consistent.